sdks/python/apache_beam/utils/windowed_value.py

```diff
@@ -80,7 +80,7 @@ def __cmp__(left, right):  # pylint: disable=no-self-argument
     on unequal values (always returning 1).
     """
     if type(left) is not type(right):
-      return cmp(type(left), type(right))
+      return 1
```

Comment:
This is a change in behaviour even though it matches the comment above. What does this affect?

Reply:
All of the tests pass. If you were toe sort a windowed value with something which was not a windowed value the ordering could change.

Reply:
Sounds good.

cc: @robertwb  since this pertains to the todo below.